articles/TOC.yml

```diff
@@ -1,33 +1,54 @@
-- name: Welcome
+- name: Quantum Development Kit Preview Documentation
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
-- name: Quantum computing concepts
-  topicHref: concepts/index.md
-  href: concepts/toc.yml
-- name: Quickstart - your first quantum program
-  href: quickstart.md
-- name: Managing quantum machines and drivers
-  topicHref: machines/index.md
-  href: machines/toc.yml
-- name: Quantum development techniques
-  topicHref: techniques/index.md
-  href: techniques/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Q# programming language
-  topicHref: language/index.md
-  href: language/toc.yml
-- name: Contributing to the Quantum Development Kit
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: Glossary
-  href: Glossary.md
-- name: For more information
-  href: for-more-info.md
-- name: Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
-- name: License
-  href: license.md
+- name: Overview
+  items:
+  - name: Getting started
+    href: install-guide/index.md
+  - name: Contributing to the Quantum Development Kit
+    href: contributing/toc.yml
+    topicHref: contributing/index.md
+- name: Quickstarts
+  items:
+  - name: C#
```

Comment:
I like separating out quickstarts like this, but at the moment it feels like a mix of install instructions (C#, Jupyter / IQ#, and Python) and of quickstarts that help users write their first Q# program without reference to the choice of development environment or host language. For example, I don't see "Create a Bell state in Q#" as an alternative task to "Python," but as something I might want to do once I have Python set up to work with Q#.

Reply:
Hi Chris - Natalie asked that I submit an initial rework of the TOC without restructuring any articles within the existing site.  So that is why the "Getting started" material is under quickstarts, even though it is mostly prerequisite material.  I have reworked the Bell State in Q# quickstart offline to have a prerequisite section that contains the necessary steps that are currently in Getting started.  Typically Azure programming articles have a prerequisite section in case a reader navigates to it directly from a Google search, etc.  So essentially what you see is an interim step, not the final vision for the TOC.

Reply:
Makes sense. We may want to re-examine the concepts in the Bell state quickstart as well, then, to be clear about the learning goals of that page. At the moment, we've treated Bell state preparation as a stand-in for a simple task one could perform as a part of a larger Q# application; it's nice from the standpoint that also demonstrates an example of entanglement, but we have definitely seen issues where people have confused "Bell state preparation" with "Bell tests" (aka CHSH games).

Promoting Bell state preparation to the title of the quickstart puts the specifics of the example ahead of the more general learning outcomes we had previously (see below), such that I agree with this being an interim step that will need some more iteration.

![image](https://user-images.githubusercontent.com/31516/64291088-3531c980-cf1c-11e9-828f-209374a58c21.png)